timemachine/fe/mle.py

```diff
@@ -212,3 +214,103 @@ def infer_node_vals_and_errs(
     dg_err = bootstrap_estimates.std(0)
 
     return dg, dg_err
+
+
+def infer_node_vals_and_errs_networkx(
+    nx_graph: nx.Graph,
+    edge_diff_prop: str,
+    edge_stddev_prop: str,
+    node_val_prop: str,
+    node_stddev_prop: str,
+    ref_nodes: Sequence[Any],
+    ref_node_val_prop: str,
+    ref_node_stddev_prop: str,
+    n_bootstrap: int = 100,
+    seed: int = 0,
+) -> nx.Graph:
+    """Version of :py:func:`timemachine.fe.mle.infer_node_vals_and_errs` that accepts a networkx graph.
+
+    Parameters
+    ----------
+    nx_graph: networkx.Graph
+        Networkx graph
+    edge_diff_prop: str
+        Edge property to use for differences
+    edge_stddev_prop: str
+        Edge property to use for standard deviations
+    node_val_prop: str
+        Node property in which to record inferred absolute values
+    node_stddev_prop: str
+        Node property in which to record inferred standard deviations
+    ref_nodes: sequence
+        Reference nodes (must have properties `ref_node_val_prop` and `ref_node_stddev_prop`)
+    ref_node_val_prop: str
+        Node property to use for reference values. Must be defined on reference nodes.
+    ref_node_stddev_prop: str
+        Node property to use for reference standard deviations. If undefined in a reference node, assumed to be zero.
+    n_bootstrap, seed:
+        See documentation for :py:func:`timemachine.fe.mle.infer_node_vals_and_errs`
+
+    Returns
+    -------
+    networkx.Graph
+        Graph where all nodes have been labeled with the inferred value of `node_val_prop` and `node_stddev_prop`.
+    """
+
+    def get_valid_subgraph(g: nx.Graph) -> nx.Graph:
+        "Remove edges with missing edge_diff_prop or edge_stddev_prop; then, remove any isolated nodes"
```

Comment:
super nit: triple quotes?

Reply:
Changed in https://github.com/proteneer/timemachine/pull/987/commits/47c8325bcebb9e218c7118a6444e2a234c728cfb